
<!-- Scenario -->
<div class="container-fluid gc-section gc-image" style="background-image: url('imgs/scenario_bg.jpg'); min-height: 27rem;">
  <div class="container">
    <section>
      <h3 class="title">Scenario</h3>
      <h4 class="subtitle">
        The human genome is a sequence of 3 billions of DNA nucleotides. Most diseases have a genetic component, hence storing, querying and integrating “big” genomic data is of paramount importance.
      </h4>
    </section>
  </div>
</div>


<div class="container gc-section">
  <section>
    <p class="text">
      Genomic computing is a new science focused on understanding the functioning of the genome, as a premise to fundamental discoveries in biology and medicine. 
      The challenge is to answer to fundamental questions for biological and clinical research, e.g. how cancer arises and develops, how driving mutations occur, how much complex disorders such as cancer depend on environmental factors or genetic predisposition; and then use individual genomic information for personalized/precision medicine.
    </p>
  </section>

  <!--Next generation sequencing-->
  <section>
    <h3 class="title">Next generation sequencing</h3>
    <h4 class="subtitle">Next Generation Sequencing (NGS) allow a fast and inexpensive technology (few hours and less than on thousand dollars) to read the whole human genome.</h4>
    <p class="text">
      Next Generation Sequencing (NGS) allows the production of the entire human genome sequence at a cost of about 1000 US $. Huge amounts of sequence data are continuously collected by a growing number of research laboratories, often organized through world-wide consortia (such as ENCODE, TCGA, the 1000 Genomes Project and the Roadmap Epigenomics Project.)
    </p>
    <div class="text">
      NGS Technology provides affordable (in time and cost) and precise determinations of genome wide:
      <ul class="p-l-1 m-t-1">
        <li class="text">DNA sequence/variations (DNA-seq)</li>
        <li class="text">Gene subregions’ activity (RNA-seq) [all gene test]</li>
        <li class="text">Protein-DNA interaction regions (ChIP-seq)</li>
      </ul>
    </div>

    <div class="row p-t-3">
      <div class="col-md-6 text-xs-center">
        <figure class="figure">
          <img class="figure-img img-fluid" src="imgs/cost-per-genome.jpg" alt="Cost per Genome" style="max-height: 300px;">
          <figcaption class="figure-caption text" style="font-size: 75%;">
            Image <a href="https://www.genome.gov/sequencingcosts/" target="_blank">source</a>
          </figcaption>
        </figure>
      </div>
      <div class="col-md-6 text-xs-center">
        <figure class="figure">
          <img class="figure-img img-fluid" src="imgs/illumina-machine.jpg" alt="Tertiary analysis" style="max-height: 300px;">
          <figcaption class="figure-caption text" style="font-size: 75%;">
            Image <a href="http://www.yourgenome.org/facts/what-is-the-illumina-method-of-dna-sequencing" target="_blank">source</a>
          </figcaption>
        </figure>
      </div>
    </div>

  </section>
  
  
  
  <!-- Tertiary data analysis -->
  <section>
    <h3 class="title">Tertiary data analysis</h3>
    <p class="text">
      NGS technology produces raw data, i.e., short reads of DNA or RNA, that are then processed in a variety of ways. First, with “primary analysis”, reads must be produced; then with “secondary analysies” data must be aligned to the reference genome, and “features” (i.e., properties of DNA or RNA regions) have to be “called” (i.e., computed by programs); finally, “tertiary analysis” occurs, focused on mastering and understanding processed data. 
    </p>
    <p class="text">
      While bio-informatics offers a huge number of different tools for ad-hoc primary and secondary analysis, with technology-driven formats and task-oriented language extensions, little emphasis ia directed towards “tertiary analysis”, which requires powerful abstractions, format-independent representations, and out-of-the box thinking and scaling. 
    </p>
    <div class="text-xs-center">
      <figure class="figure">
        <img class="figure-img img-fluid" src="imgs/tertiary.jpg" alt="Tertiary analysis" style="max-height: 300px;">
        <figcaption class="figure-caption text" style="font-size: 75%;">
          Image <a href="http://blog.goldenhelix.com/grudy/a-hitchhikers-guide-to-next-generation-sequencing-part-2/" target="_blank">source</a>
        </figcaption>
      </figure>
    </div>
  </section>


</div>