<!--video-->
<div class="container-fluid gc-section">
    <div class="container">


        Inspired by our work on genomic data integration, after the outbreak of the COVID-19 pandemic we turned our
        attention to the analysis and management of viral sequences, aiming at re-applying the data-driven paradigm used
        for human genomics.

        <br><br>
        The GeCo approach to the domain of viral genomics presents many analogies with our previous approach to human
        genomics.
        <br><br>
        We observed the need for data integration across heterogeneous sources collecting viral sequences, we thus
        modeled a consensus conceptual schema for both metadata and sequence properties (mutations of viral sequences
        resulting into changes of the amino-acids forming proteins) and we built search systems over sequence
        repositories, with powerful data visualization tools.
        <br><br>

        Results include:
        <br>
        <ul>
            <li>the Viral Conceptual Model (VCM); describing sequences with their annotations, nucleotide mutations and
                amino
                acid changes; aspects related to the technology, biology, and organization of viral sequencing are also
                reported. The model
                has set the basis for a rich stream of research that applies data-driven methods to practically support
                virology
                research.
            </li>
            <li>Within a large data warehouse, we integrate sequences with their metadata from a variety of different
                sources;
                we make it searchable through the ViruSurf (<a href="http://www.gmql.eu/virusurf/" target="_blank"
                                                               rel="noopener noreferrer">http://www.gmql.eu/virusurf/</a>)
                search interface, supporting the
                extraction of sequences based on their combined mutations, to compare different conditions, and to build
                interesting populations for downstream analysis.
            </li>
            <li>To such search system, we associate a powerful data visualization application called VirusViz
                (<a href="http://www.gmql.eu/virusviz/" target="_blank" rel="noopener noreferrer">http://www.gmql.eu/virusviz/</a>),
                which receives the
                results of queries and provides metadata summarization,
                variant descriptions, and variant visualization. VirusViz offers a rich customization and the
                possibility of
                grouping and comparatively analyzing populations of interest (allowing, for instance, to trace new
                variants
                since their starting dates in a precise location).
            </li>
            <li>EpiSurf (<a href="http://www.gmql.eu/episurf/" target="_blank" rel="noopener noreferrer">http://www.gmql.eu/episurf/</a>)
                is a search engine for selecting sequences and epitopes and integrating them for testing epitope
                stability.
            </li>
            <li>ViruClust (<a href="http://www.gmql.eu/viruclust/" target="_blank" rel="noopener noreferrer">http://www.gmql.eu/viruclust/</a>
                and <a href="https://github.com/DEIB-GECO/Docker_ViruClust" target="_blank" rel="noopener noreferrer">https://github.com/DEIB-GECO/Docker_ViruClust/</a>)
                is tool for fine tuning of clusters of sequences, used for fine-grain surveillance.
            </li>
            <li>VariantHunter (<a href="http://gmql.eu/variant_hunter/" target="_blank"
                                  rel="noopener noreferrer">http://gmql.eu/variant_hunter/</a>)
                is an application that monitors the evolution of mutations and indicates possible emerging variants.
            </li>
            <li>CoV2K is an abstract model integrating data and knowledge aspects of SARS-CoV-2 (see API documentation
                at
                <a href="http://gmql.eu/cov2k/api/" target="_blank" rel="noopener noreferrer">http://gmql.eu/cov2k/api/</a>).
            </li>
        </ul>
        Other systems are currently under development.
        <br><br>

        <div class="container-fluid gc-section gc-full gc-image"
             style="background-image: url('imgs/schema_tools.png'); background-size: contain; min-height: 40rem"></div>

        <br><br>

        <h3 style="text-align: center">GeCo Team working on Viral Genomics.</h3>
        <br>
        <div class="container-fluid gc-section gc-full gc-image"
             style="background-image: url('imgs/whole-team-viral2.png'); background-size: contain; min-height: 40rem"></div>


        <br><br>
        <h3>VIRAL GENOMICS PUBLICATIONS</h3>

        <ul class="publications">
            <!--SWAT4LS poster-->

            <!--CoV2k-->
            <li>
                <em>Alfonsi, T., Al Khalaf, R., Ceri, S., Bernasconi, A.</em>
                <strong>
                    CoV2K model, a comprehensive representation of SARS-CoV-2 knowledge and data interplay.</strong>
                Accepted for publication in the Scientific Data Journal [in print]
                <!--<a target="_blank" href="https://doi.org/10.3390/biomed2010007">Publisher's link</a> --->
                <a target="_blank" class="pdf" href="publications/CoV2K_Article.pdf">pdf</a>
            </li>

            <!--BioTech Pipeline-->
            <li>
                <em>Alfonsi, T., Pinoli, P., Canakoglu, A.</em>
                <strong>High Performance Integration Pipeline for Viral and Epitope Sequences.</strong>
                BioTech 2022, 11(1), 7.
                DOI: 10.3390/biotech11010007
                <a target="_blank" href="https://doi.org/10.3390/biotech11010007">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/biotech-11-00007.pdf">pdf</a>
            </li>

            <!--BioMed-->
            <li>
                <em>Bernasconi, A., Ceri, S.</em>
                <strong>Interoperability of COVID-19 Clinical Phenotype Data with Host and Viral Genetics Data.</strong>
                BioTech 2022, 2(1), 69-81.
                DOI: 10.3390/biomed2010007
                <a target="_blank" href="https://doi.org/10.3390/biomed2010007">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/biomed-02-00007.pdf">pdf</a>
            </li>

            <!--ViruClust-->
            <li>
                <em>Cilibrasi, L., Pinoli, P., Bernasconi, A., Canakoglu, A., Chiara, M., Ceri, S.</em>
                <strong>ViruClust: direct comparison of SARS-CoV-2 genomes and genetic variants in space and
                    time.</strong>
                Bioinformatics (2022)
                DOI: 10.1093/bioinformatics/btac030
                <a target="_blank" href="https://doi.org/10.1093/bioinformatics/btac030">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/ViruClust_preprint.pdf">pdf</a>
            </li>

            <!--microarray-->
            <li>
                <em>Bernasconi, A., Cascianelli, S.</em>
                <strong>Scenarios for the Integration of Microarray Gene Expression Profiles in COVID-19–Related
                    Studies.</strong>
                In: Agapito G. (eds) Microarray Data Analysis. Methods in Molecular Biology, vol 2401, 2022. Humana, New
                York, NY.
                DOI: 10.1007/978-1-0716-1839-4_13
                <a target="_blank" href="https://doi.org/10.1007/978-1-0716-1839-4_13">Publisher's link</a>
                <!---
                <a target="_blank" class="pdf" href="publications/biotech-10-00027.pdf">pdf</a>-->
            </li>

            <!--VirusLab-->
            <li>
                <em>Pinoli, P., Bernasconi, A., Sandionigi, A., Ceri, S.</em>
                <strong>VirusLab: a tool for customized SARS-CoV-2 data analysis.</strong>
                BioTech 2021, 10(4), 27.
                DOI: 10.3390/biotech10040027
                <a target="_blank" href="https://www.mdpi.com/2673-6284/10/4/27">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/biotech-10-00027.pdf">pdf</a>
            </li>

            <!--CoVolution-->
            <li>
                <em>Bernasconi, A., Mari, L., Casagrandi, R., Ceri, S.</em>
                <strong>Data-driven analysis of amino acid change dynamics timely reveals SARS-CoV-2 variant
                    emergence.</strong>
                Scientific Reports 11, 21068 (2021).
                DOI: 10.1038/s41598-021-00496-z
                <a target="_blank" href="https://doi.org/10.1038/s41598-021-00496-z">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/CoVolution_Scientific_Reports.pdf">pdf</a>
            </li>

            <!--EpiSurf Database-->
            <li>
                <em>Bernasconi, A., Cilibrasi, L., Al Khalaf, R., Alfonsi, T., Ceri, S., Pinoli, P., Canakoglu, A. </em>
                <strong> EpiSurf: metadata-driven search server for analyzing amino acid changes on epitopes of
                    SARS-CoV-2 and other viral species</strong>
                Database, Volume 2021, 2021, baab059.
                DOI: 10.1093/database/baab059
                <a target="_blank" href="https://doi.org/10.1093/database/baab059">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/baab059.pdf">pdf</a>
            </li>


            <!--ER 2021-->
            <li>
                <em>Guizzardi G., Bernasconi A., Pastor O., Storey V.</em>
                <strong>Ontological Unpacking as Explanation: The Case of the Viral Conceptual Model.</strong>
                In: Ghose A., Horkoff J., Silva Souza V.E., Parsons J., Evermann J. (eds) Conceptual Modeling. ER 2021.
                Lecture Notes in Computer Science, vol 13011. Springer, Cham.
                DOI: 10.1007/978-3-030-89022-3_28
                <a target="_blank" href="https://doi.org/10.1007/978-3-030-89022-3_28">Publisher's link</a>
                -
                <a target="_blank" class="pdf"
                   href="publications/Ontological_Unpacking_as_Explanation_The_Case_of_the_ViralConceptual_Model.pdf">pdf</a>
            </li>


            <!--VIRUSVIZ2021 NAR methods-->
            <li>
                <em>Bernasconi A., Gulino A., Alfonsi T., Canakoglu A., Pinoli P., Sandionigi A., Ceri S.</em>
                <strong>VirusViz: Comparative analysis and effective visualization of viral nucleotide and amino acid
                    variants</strong>
                Nucleic Acids Research, Volume 49, Issue 15, 7 September 2021, Page e90.
                DOI: 10.1093/nar/gkab478
                <a target="_blank" href="https://doi.org/10.1093/nar/gkab478">Publisher's
                    link</a>
                -
                <a target="_blank" class="pdf" href="publications/gkab478-virusviz.pdf">pdf</a>
            </li>

            <!--RCIS2021-->
            <li>
                <em>Al Khalaf R., Alfonsi T., Ceri S., Bernasconi A.</em>
                <strong>CoV2K: a Knowledge Base of SARS-CoV-2 Variant Impacts</strong>
                In: Cherfi S., Perini A., Nurcan S. (eds) Research Challenges in Information Science. RCIS 2021. Lecture
                Notes in Business Information Processing, vol 415. Springer, Cham.
                DOI: 10.1007/978-3-030-75018-3_18
                <a target="_blank" href="https://link.springer.com/chapter/10.1007/978-3-030-75018-3_18">Publisher's
                    link</a>
                -
                <a target="_blank" class="pdf" href="publications/paper_84-RCIS2021.pdf">pdf</a>
            </li>

            <!--REFSQ2021-->
            <li>
                <em>Bernasconi A.</em>
                <strong>Extreme Requirements Elicitation: Lessons Learnt from the COVID-19 Case Study</strong>
                In the Joint Proceedings of the Co-Located Events of the International Working Conference on Requirement
                Engineering: Foundation for Software Quality (REFSQ2021). CEUR Workshop Proceedings 2857, CEUR-WS.org
                2021.
                <a target="_blank" href="http://ceur-ws.org/Vol-2857/">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/Poster-68-REFSQ2021.pdf">pdf</a>
            </li>

            <!--VIRUSURF-->
            <li>
                <em>Canakoglu A., Pinoli P., Bernasconi A., Alfonsi T., Melidis D.P., Ceri S.</em>
                <strong>ViruSurf: an integrated database to investigate viral sequences</strong>
                Nucleic Acids Research, Volume 49, Issue D1, 8 January 2021, Pages D817–D824.
                DOI: 10.1093/nar/gkaa846
                <a target="_blank" href="https://academic.oup.com/nar/article/49/D1/D817/5921283">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/ViruSurf_NAR_Database_Issue___REVISION.pdf">pdf</a>
            </li>

            <!--BIB COVID19-->
            <li>
                <em>Bernasconi A., Canakoglu A., Masseroli M., Pinoli P., Ceri S.</em>
                <strong>A review on viral data sources and search systems for perspective mitigation of
                    COVID-19</strong>
                Briefings in Bioinformatics, Volume 22, Issue 2, March 2021, Pages 664–675.
                DOI: 10.1093/bib/bbaa359.
                <a target="_blank" href="https://doi.org/10.1093/bib/bbaa359">Publisher's link</a>
                -
                <a target="_blank" class="pdf" href="publications/covid19_briefings.pdf">pdf</a>
            </li>

            <!--ER 2020-->
            <li>
                <em>Bernasconi A., Canakoglu A., Pinoli P., Ceri S.</em>
                <strong>Empowering Virus Sequence Research through Conceptual Modeling</strong>
                In: Dobbie G., Frank U., Kappel G., Liddle S.W., Mayr H.C. (eds) Conceptual Modeling. ER 2020. Lecture
                Notes
                in Computer Science, vol 12400. Springer, Cham.
                DOI: 10.1007/978-3-030-62522-1_29
                <a target="_blank" href="https://doi.org/10.1007/978-3-030-62522-1_29">Publisher's link</a>
                -
                <a target="_blank" class="pdf"
                   href="publications/Viral_Conceptual_Model___ER2020_GeCoWebsite.pdf">pdf</a>
            </li>

        </ul>


        <br><br>
        <h3>CONSORTIA PAPER</h3>
        We also collaborated with the <a target="_blank" href="https://www.covid19hg.org/">COVID-19 Host Genetics
        Initiative</a>,
        producing the COVID-19 phenotype data dictionary, available at <a target="_blank"
                                                                          href="http://gmql.eu/phenotype/">http://gmql.eu/phenotype/</a>.
        Here follows a list of the papers where we were involved within the Initiative.
        <ul class="publications">

            <li>
                <em>Daga, S., et al. </em>
                <strong>Employing a systematic approach to biobanking and analyzing clinical and genetic data for
                    advancing COVID-19 research.</strong>
                European Journal of Human Genetics 2021, 29(5), pp.745-759.
                DOI: 10.1038/s41431-020-00793-7
                <a target="_blank" href="https://doi.org/10.1038/s41431-020-00793-7">Publisher's link</a>
            </li>

            <li>
                <em>COVID-19 Host Genetics Initiative</em>
                <strong>Mapping the human genetic architecture of COVID-19.</strong>
                Nature 2021, 600(7889), pp.472-477.
                DOI: 10.1038/s41586-021-03767-x
                <a target="_blank" href="https://doi.org/10.1038/s41586-021-03767-x">Publisher's link</a>
            </li>

            <li>
                <em>Van Blokland, I.V. et al.</em>
                <strong>Using symptom-based case predictions to identify host genetic factors that contribute to
                    COVID-19 susceptibility.</strong>
                PloS one 2021, 16(8), p.e0255402.
                DOI: 10.1371/journal.pone.0255402
                <a target="_blank" href="https://doi.org/0.1371/journal.pone.0255402">Publisher's link</a>
            </li>


        </ul>


        <!-- forum -->
        <br><br>
        <h3>Blog posts</h3>

        <div class="name">Virological.org (posted on Nov. 30th, 2021)</div>
        <div class="where text">Anna Bernasconi, Pietro Pinoli, Ruba Al Khalaf, Tommaso Alfonsi, Arif Canakoglu, Luca
            Cilibrasi, and Stefano Ceri.
        </div>
        <div class="text font-weight-bold"> Report on Omicron Spike mutations on epitopes and
            immunological/epidemiological/kinetics effects from literature -
            <a target="_blank"
               href="https://virological.org/t/report-on-omicron-spike-mutations-on-epitopes-and-immunological-epidemiological-kinetics-effects-from-literature/770">Link</a>
        </div>

        <br><br>

        <div class="name">Virological.org (posted on May 11th, 2022)</div>
        <div class="where text">Matteo Chiara, Pietro Pinoli, Luca Minotti, Anna Bernasconi, Arif Canakoglu, Erica
            Ferrandi, and Stefano Ceri.
        </div>
        <div class="text font-weight-bold"> Using VariantHunter for detecting emerging SARS-CoV-2 variants -
            <a target="_blank"
               href="https://virological.org/t/using-varianthunter-for-detecting-emerging-sars-cov-2-variants/797">Link</a>
        </div>


        <!-- Tutorials -->
        <br><br>
        <h3>SEMINARS AND TUTORIALS</h3>

        <div class="name">EDBT 2022 Tutorial</div>
        <div class="text font-weight-bold"> Conceptual models and databases for searching the genome
        </div>
        <div class="where text">Speakers: Anna Bernasconi, Pietro Pinoli</div>
        <div class="where text">April 1st, 2022</div>
        <div class="website text">
            <a target="_blank"
               href="https://www.youtube.com/watch?v=HN6XSxCt1qU&list=PL-d3yqkWNfo9chJQE1wpGmKFzmd_DBUWF">Presentation
                Video Playlist</a> -
            <a target="_blank" href="https://openproceedings.org/2022/conf/edbt/tutorial-2.pdf">Tutorial Paper</a>
        </div>
        <br><br>

        <div class="name">Seminar at the Department of Electronics, Information, and Bioengineering @ Politecnico di
            Milano
        </div>
        <div class="text font-weight-bold">Data-driven SARS-CoV-2 understanding and hunting(searching for the new
            Omicron)
            <div class="where text">Speaker: Anna Bernasconi</div>
        </div>
        <div class="where text">January 14th, 2022</div>
        <div class="website text">
            <a target="_blank" href="https://youtu.be/VHgb1Dz00ho">Video</a> -
            <a target="_blank" href="presentations/2022-01-14-Anna_Bernasconi_seminarioDeib.pdf">Slides</a>
        </div>
        <br><br>
        <div class="name">ER 2021 Tutorial</div>
        <div class="text font-weight-bold"> Conceptual models for searching the genome
        </div>
        <div class="where text">Speakers: Anna Bernasconi, Pietro Pinoli</div>
        <div class="where text">October 20th, 2021</div>
        <div class="website text">
            <a target="_blank" href="https://youtu.be/JN8xZSd2r2M">Presentation Video </a> -
            <a target="_blank" href="https://youtu.be/3WrnkFVRzkQ">Interactive Session Video </a>
        </div>
        <br><br>
        <div class="name">Seminar at the Department of Electronics, Information, and Bioengineering @ Politecnico di
            Milano for Informatics Europe
        </div>
        <div class="text font-weight-bold">Storing and Analyzing Viral Sequences through Data-driven Genomic Computing
            (What you always wanted to know about the COVID-19 virus and never dared to ask)
            <div class="where text">Speaker: Stefano Ceri</div>
        </div>
        <div class="where text">September 22nd, 2021</div>
        <div class="website text">
            <a target="_blank" href="https://www.informatics-europe.org/activities/webinars.html#sep2021">Event link</a>
            -
            <a target="_blank" href="https://youtu.be/YQibGOphGko">Video</a> -
            <a target="_blank" href="https://www.informatics-europe.org/images/webinars/IEWebinarSlides_Sep2021.pdf">Slides</a>
        </div>
        <br><br>
        <div class="name">Seminar at the Italian Symposium on Advanced Database systems (SEBD)</div>
        <div class="text font-weight-bold">Storing and Analyzing Viral Sequences through Data-driven Genomic Computing
            (What you always wanted to know about the COVID-19 virus and never dared to ask)
            <div class="where text">Speaker: Stefano Ceri</div>
        </div>
        <div class="where text">September 6th, 2021</div>
        <div class="website text">
            <a target="_blank" href="https://events.dimes.unical.it/sebd2021/invited_talks.php">Event link</a>
        </div>


    </div>

</div>

</div>